web/src/ChooseArea.svelte

```diff
@@ -93,67 +111,71 @@
   </div>
 
   <div class="map">
-    <div style="position: relative; width: 100%; height: 100%;">
-      <MapLibre
-        style={`https://api.maptiler.com/maps/streets-v2/style.json?key=${maptilerApiKey}`}
-        standardControls
-        bounds={[-8.943, 54.631, -0.901, 59.489]}
-        on:error={(e) => {
-          // @ts-expect-error ErrorEvent isn't exported
-          console.log(e.detail.error);
-        }}
-      >
-        <GeoJSON data={gj} generateId>
-          <FillLayer
-            paint={{
-              "fill-color": "rgb(200, 100, 240)",
-              "fill-outline-color": "rgb(200, 100, 240)",
-              "fill-opacity": hoverStateFilter(0.0, 0.5),
-            }}
-            beforeId="Road labels"
-            manageHoverState
-            hoverCursor="pointer"
-            on:click={onClick}
-          >
-            <Popup openOn="hover" let:props>
-              <p>{props.name}</p>
-            </Popup>
-          </FillLayer>
-
-          <LineLayer
-            paint={{
-              "line-color": "rgb(200, 100, 240)",
-              "line-width": 2.5,
-            }}
-            beforeId="Road labels"
-          />
-        </GeoJSON>
-      </MapLibre>
-    </div>
+    <MapLibre
+      style={`https://api.maptiler.com/maps/streets-v2/style.json?key=${maptilerApiKey}`}
+      bounds={[-8.943, 54.631, -0.901, 59.489]}
+      on:error={(e) => {
+        // @ts-expect-error ErrorEvent isn't exported
+        console.log(e.detail.error);
+      }}
+    >
```

Comment:
Actually the other nested div was unnecessary, maybe because the map is now filling the whole page